CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## 0.19.0 / 2018-04-25
+
+* [FEATURE] Allow specifying additional Prometheus scrape configs via secret (#1246)
+* [FEATURE] Enable Thanos sidecar (#1219)
+* [FEATURE] Make AM log level configurable (#1192)
+* [FEATURE] Transition kube-prometheus to jsonnet (#1206)
+* [ENHANCEMENT] Enable Prometheus to select Service Monitors outside own namespace (#1227)
+* [ENHANCEMENT] Enrich Prometheus operator CRD registration error handling (#1208)
+* [ENHANCEMENT] Update to latest grafana dashboards in kube-prometheus (#1238)
```

Comment:
Should kube-prometheus changes be mentioned here? @brancz

Reply:
Kube prometheus is entirely unversioned and experimental, things may change at any point in time without notice. We should drop this as otherwise it may signal more stability than we want to signal.